src/components/DpStickyElement/DpStickyElement.stories.mdx

```diff
@@ -0,0 +1,44 @@
+import { Meta } from '@storybook/addon-docs'
+import DpStickyElement from './DpStickyElement'
+
+<Meta
+    title="Components/StickyElement"
+    component={DpStickyElement}
+/>
+
+# StickyElement
+
+The StickyElement component to add static position any container, menu or slide in the (top or bottom)
+of Page when scrolling.
+
+## Props:
```

Comment:
Some general thoughts on the documentation of component props, and which content should go here (in the *.stories.mdx page):

As we're already documenting the props via DocBlocks in the actual component code, and these DocBlocks are rendered in the controls panel of the actual story page, i think it is not needed to list any prop here, again. We should rather focus on improving existing DocBlocks and filling in those which are not already existing. That being said i'd opt to remove props documentation in the *.stories.mdx pages completely.

Instead, we may provide a high level description on what the component does, and in which cases to use it. The Docs page may further feature a basic example for those who are not willing to dive into the actual stories.

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/491/commits/b2964dc76dda8a6816229b387d45615c4ee4cadd